parser/wasm_parse/mod.rs

```diff
@@ -195,42 +284,45 @@ impl<'a> Parse<'a> for wasmparser::ModuleReader<'a> {
                             wasmparser::ImportSectionEntryType::Global(_) => {
                                 indices.globals.push(id);
                             }
+                            wasmparser::ImportSectionEntryType::Event(_) => {} // @@@
```

Comment:
Now it seems to only count functions and globals and that's fine. Let's just remove the comments tho.